docs/src/main/paradox/bootstrap/details.md

```diff
@@ -65,10 +65,16 @@ The illustration below may be of help in visualising this process:
 
 ![project structure](../images/bootstrap-forming-cluster.png)
 
-Additionally, a setting is provided, `akka.management.cluster.bootstrap.form-new-cluster` that can be disabled to only allow the
-node to join existing clusters (Case 2 below). This can be used to provide additional safety during the typical deployment
-as it is relatively rare that no cluster exists. It can be specified in your `application.conf` or provided as an
-argument to the process itself via `-Dakka.management.cluster.bootstrap.form-new-cluster=<true|false>`. By default, it is enabled.
+Cluster Bootstrap will always attempt to join an existing cluster if possible. However if no contact points are found during service discovery, a
+new cluster will be formed. A setting is provided, `akka.management.cluster.bootstrap.new-cluster-enabled` that can be disabled to only allow the
```

Comment:
Maybe "a new cluster will be formed by default." It's implied by the next sentence, but it might be a little bit more clear for some readers.

Reply:
👍